static/js/containers/ProfilePage.js

```diff
@@ -20,13 +20,19 @@ import {
 } from '../constants';
 import { createActionHelper } from '../lib/redux';
 import type { Profile } from '../flow/profileTypes';
+import { startProfileEdit } from '../actions/profile';
 
 class ProfilePage extends ProfileFormContainer {
   currentStep: Function = (): string => {
     const { ui: { profileStep } } = this.props;
     return profileStep;
   };
 
+  componentDidMount() {
+    const { dispatch } = this.props;
+    dispatch(startProfileEdit(SETTINGS.user.username));
```

Comment:
It's needed here to get around some UI edge cases, I can probably explain better outside of a comment.

It's not necessary on the other components because we're opening dialogs, and we dispatch `startProfileEdit` when we open the dialog.